providers/os/provider/provider.go

```diff
@@ -325,6 +325,9 @@ func (s *Service) connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 			asset.Name = fingerprint.Name
 			asset.PlatformIds = fingerprint.PlatformIDs
 		}
+		if asset.Name == "" && len(asset.Connections) > 0 {
+			asset.Name = asset.Connections[0].Host
```

Comment:
When checking this in debug on my local system, `asset.Connections[0].Host` is empty.